catroid/src/main/java/org/catrobat/catroid/ui/recyclerview/fragment/DataListFragment.kt

```diff
@@ -412,26 +428,25 @@ class DataListFragment : Fragment(),
         }
     }
 
-    private fun showDeleteAlert(selectedItems: List<UserData<*>>) {
-        AlertDialog.Builder(requireContext())
-            .setTitle(R.string.deletion_alert_title)
-            .setMessage(R.string.deletion_alert_text)
-            .setPositiveButton(R.string.delete) { _: DialogInterface?, _: Int ->
-                deleteItems(selectedItems)
-            }
-            .setNegativeButton(R.string.cancel, null)
-            .setCancelable(false)
-            .show()
-    }
-
     private fun deleteItems(selectedItems: List<UserData<*>>) {
         finishActionMode()
         for (item in selectedItems) {
             adapter?.remove(item)
         }
-        ProjectManager.getInstance().currentProject.deselectElements(selectedItems)
-        ToastUtil.showSuccess(activity, resources.getQuantityString(R.plurals.deleted_Items,
-                                                                    selectedItems.size, selectedItems.size))
+        projectManager.currentProject.deselectElements(selectedItems)
+        showSuccess(selectedItems.size)
+    }
+
+    @VisibleForTesting
+    open fun showSuccess(size: Int?) {
```

Comment:
```suggestion
    fun showSuccess(size: Int?) {
```

Since this is a final class, the open has no effect on the methods. 
please also remove it on the other methods you added.